event_waiting_list/data/website_form.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <function model="ir.model.fields" name="formbuilder_whitelist">
+        <value>crm.lead</value>
+        <value eval="[
+            'event_id',
```

Comment:
no other fields?

Reply:
In our case, other required fields are already added here : https://github.com/odoo/odoo/blob/11.0/addons/website_crm/data/website_crm_data.xml#L11

Reply:
oh right, and the function preserves existing fields. Can you add a comment here anyway? Like "additional fields are already enabled by website_crm"